Content.Server/Database/ServerDbBase.cs

```diff
@@ -1016,6 +1016,79 @@ public async Task<int> CountAdminLogs(int round)
             await using var db = await GetDb();
             return await db.DbContext.AdminLog.CountAsync(log => log.RoundId == round);
         }
+        #endregion
+
+        #region Support Logging
+
+        /// <summary>
+        /// Retrieves a SupportExchange based on the round number, target user (Guid), and server id.
+        /// </summary>
+        public async Task<SupportExchange?> GetSupportExchangeAsync(int supportRound, Guid supportTarget)
+        {
+            await using var db = await GetDb();
+            return await db.DbContext.SupportExchanges
+                .Include(e => e.SupportMessages)
+                .SingleOrDefaultAsync(e => e.SupportRound == supportRound &&
+                                          e.SupportTargetPlayer == supportTarget);
+        }
```

Comment:
You cannot outright return the `SupportExchange` object, as the database time values are not consistent between EF Core and SQLite. You need to convert them with `NormalizeDatabaseTime`.